### PR TITLE
More helpful info on yarn upgrade

### PR DESCRIPTION
**Pull request by lukasoppermann**
Showing my dependencies first when running `yarn upgrade`.

@bestander hope this is better.

### CODE REVIEW

File: 0000-template.md [hunk 1]
```diff
@@ -1,54 +0,0 @@
-- Start Date: (fill me in with today's date, YYYY-MM-DD)
```
Comment (bestander) [inline 0000-template.md]: I think it is better to keep template in the repo

File: 0000-show-updated-packages-only.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# Detailed design
+
+Instead of just showing all dependencies the following should be shown.
+
+```
+success Saved lockfile.    
+success Saved 774 new dependencies.    
+
+Your updated dependencies:
```
Comment (bestander) [inline 0000-show-updated-packages-only.md]: Maybe "direct dependencies?"

**Comment by bestander** [comment]
Looks useful.
Just a few nits and let's get this in.

**Comment by lukasoppermann** [comment]
Hey @bestander, thanks fir the input, changed it.

**Comment by bestander** [comment]
Send a PR :)

**Comment by lukasoppermann** [comment]
@bestander I have no idea how this can be achieved, sorry. :(

**Comment by bestander** [comment]
@lukasoppermann, if your problem is that you are not familiar with the code I'll be happy to assist you and guide through a PR.
Feel free to ping me on an empty PR or in discord channel

**Pull request merged**